[2024-07-08T17:20:18.250+0000] {scheduler_job_runner.py:769} ERROR - Executor reports task instance <TaskInstance: fetch_and_store_amazon_books.insert_book_data scheduled__2024-06-28T00:00:00+00:00 [queued]> finished (failed) although the task says it's queued. (Info: None) Was the task killed externally?
